pkg/models/move_task_order.go

```diff
@@ -35,6 +35,8 @@ type MoveTaskOrder struct {
 	PrimeEstimatedWeight             *unit.Pound         `db:"prime_estimated_weight"`
 	PrimeEstimatedWeightRecordedDate *time.Time          `db:"prime_estimated_weight_recorded_date"`
 	RequestedPickupDate              time.Time           `db:"requested_pickup_date"`
+	SubmittedCounselingInfoDate      time.Time           `db:"submitted_counseling_info_date"`
+	AvailableToPrimeDate             time.Time           `db:"available_to_prime_date"`
```

Comment:
Can these be null? The migration makes me think that they can and so these ought to be changed.

```suggestion
	SubmittedCounselingInfoDate             *time.Time           `db:"submitted_counseling_info_date"`
	AvailableToPrimeDate             *time.Time           `db:"available_to_prime_date"`
```

If you don't want them to be null then you need a migration that firsts creates the columns and sets the values to nil and then you come back later and modify them to `NOT NULL`. But in the current state I think you'll get nil pointer dereferencing.

Reply:
You are correct. I'll fix them to be nullable.